### PR TITLE
Updated logger print and post-build events fixes

### PR DESCRIPTION
**Pull request by Chupakabra0**

### CODE REVIEW

File: utils/include/CResponseHandler.cpp [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+EFrameError CResponseHandler::GetErrorCodeFromFrame(
+    const std::string& response_str)
+{
+    nlohmann::json response = nlohmann::json::parse(response_str);
+    int err = response[GlobalVariable::c_frame_error].get<int>();
+    return EFrameError(err);
+}
+
+bool CResponseHandler::TryValidate(const std::string& guid,
```
Comment (AntonVoytov) [inline utils/include/CResponseHandler.cpp:50]: This method should be reworked

File: utils/include/CRequestHandler.cpp [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+EFrameError CRequestHandler::GetErrorCodeFromFrame(
+    const std::string& request_str)
+{
+    nlohmann::json request = nlohmann::json::parse(request_str);
+    int err = request[GlobalVariable::c_frame_error].get<int>();
+    return EFrameError(err);
+}
+
+bool CRequestHandler::TryValidateRequest(const std::string& request_str)
```
Comment (AntonVoytov) [inline utils/include/CRequestHandler.cpp:116]: Should be reworked

File: utils/include/CRequestHandler.cpp [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+                error_from_frame);
+            //log and return true
+            CLOG_ERROR("Invalid request from the client!!!");
+            return false;
+        }
+
+        switch (AnalyzeRequestType(request))
+        {
+        case ERequestType::ALL_DATA:
+            if (!ExecuteRequest(answer, std::make_shared<CRequestExcAllData>(
```
Comment (AntonVoytov) [inline utils/include/CRequestHandler.cpp:56]: Create a temporary variable to create a request object in "switch case".
ExecuteRequest(answer, "temporary") after switch case

**Review by AntonVoytov — COMMENTED** [review]
Please review coments

File: utils/include/CResponseHandler.cpp (hunk 1 above)
Comment (Crowy-o) [reply utils/include/CResponseHandler.cpp:50]: Added more sense to this method: now it's not only checking key's name, but also check is frame store something beside keys' names that request shouldn't store

File: utils/include/CRequestHandler.cpp (hunk 2 above)
Comment (Crowy-o) [reply utils/include/CRequestHandler.cpp:117]: Added more sense to this method: now it's not only checking key's name, but also check is frame store something beside keys' names that request shouldn't store

File: utils/include/CRequestHandler.cpp (hunk 3 above)
Comment (Crowy-o) [reply utils/include/CRequestHandler.cpp]: Changed

**Pull request merged**